### ISSUE TITLE
wsl2 maybe conflict with AMD new ryzen CPU/GPU,making screen splashing randomly

### ISSUE THREAD

**Issue by emacsenli** [microsoft/WSL2-Linux-Kernel#166]
hey guys I  have bought a new asus notebook with amd 3th  Ryzen7 4700U CPU and 16GB memories inside of it 

winver : OS 2004 19041.388 home edition

all is ok without enabling wsl2 , then according to this official quickstart  https://docs.microsoft.com/en-us/windows/wsl/install-win10 , I enabled wsl2 and after restarted and re-entered win10 ,screen begin to splash randomly  as if I was blinking my eyes ,even external screen with HDMI splashes too at the same time

there is no way to stop it and the only thing I can do is to restore win10 entirely 

so is  this  a bug of wsl2 or something wrong with amd GPU driver ?

thanks in advance

**Comment by rekaerst** [comment]
Have you tried to update your display adapter? 

I have a Lenovo laptop with Ryzen 5 4600H CPU from the same renoir series and 16GB of ram
winver: OS 2004 19041.388 home edition
my display driver version is 27.20.1017.1011

**Comment by emacsenli** [comment]
> Have you tried to update your display adapter?
> 
> I have a Lenovo laptop with Ryzen 5 4600H CPU from the same renoir series and 16GB of ram
> winver: OS 2004 19041.388 home edition
> my display driver version is 27.20.1017.1011

No I've  updated to same driver as you, but problem is  still there.

I have noticed once after enabling wsl2 ,and the very first seconds of relogin into win10 ,there would be a splash of an  unknown window ,which seemed just want pop out but failed . then  screen began to splash .

so this may not be a driver problem ,but someting else conflicting.

any ideas to identify this problem WILL be great .thanks in advance AGAIN

**Comment by rekaerst** [comment]
Can you login to your linux distro's shell?

**Comment by emacsenli** [comment]
> Can you login to your linux distro's shell?

yes I can log into wsl2 distros and  all at least working perfect  by now

**Comment by ameeno** [comment]
sounds like a hardware issue. I use a ideapad 5 15are05  with a 4800U cpu and all is well. 

I did need lenovo to come out and change the motherboard because it was faulty.
